eng/Packages.Data.props

```diff
@@ -182,10 +182,14 @@
 	
     <PackageReference Update="Microsoft.Azure.Devices.Client" Version="1.27.0" />
 	
-	 <!-- Mgmt sdk packages-->
+    <!-- Mgmt sdk packages-->
     <PackageReference Update="Azure.ResourceManager.Resources" Version="1.0.0-preview.2" />
     <PackageReference Update="Azure.ResourceManager.Compute" Version="1.0.0-preview.2" />
     <PackageReference Update="Azure.ResourceManager.Network" Version="1.0.0-preview.2" />
     <PackageReference Update="Azure.ResourceManager.Storage" Version="1.0.0-preview.2" />
+
+    <!-- Perf/Stress packages-->
```

Comment:
This sets the version but I didn't see you actually reference these package in your stress project.

Reply:
They are only needed by the stress framework.